lib/jira/resource/agile.rb

```diff
@@ -31,6 +31,18 @@ def self.get_sprint_issues(client, sprint_id, options = {})
         parse_json(response.body)
       end
 
+      def self.get_projects_full(client, board_id, options = {})
+        response = client.get("/rest/agile/1.0/board/#{board_id}/project/full")
```

Comment:
You should use `path_base(client)` instead of `/rest/agile/1.0` (as used in the other methods)